digits/model/tasks/caffe_train.py

```diff
@@ -703,7 +703,11 @@ def task_arguments(self, resources, env):
             if len(identifiers) == 1:
                 args.append('--gpu=%s' % identifiers[0])
             elif len(identifiers) > 1:
-                args.append('--gpus=%s' % ','.join(identifiers))
+                if config_value('caffe_root')['version'] < utils.parse_version('0.13.99'):
```

Comment:
would this work too: 
if config_value('caffe_root')['version'] < utils.parse_version('0.14')
I don't suppose the odds of seeing a 0.13.991 version are high but since you're using a strict inequality, why not compare against 0.14?